### PR TITLE
Search field correct values

### PR DESCRIPTION
**Pull request by 4etverochka**

### CODE REVIEW

File: src/main/java/constants/ResultValues.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+            "Printed Chiffon Dress", "Printed Dress");
+    public static final List<String> T_SHIRTS_RESULT = Collections.singletonList("Faded Short Sleeve T-shirts");
+    public static final List<String> CASUAL_DRESSES_RESULT = asList("Printed Summer Dress", "Printed Dress");
+    public static final List<String> SUMMER_DRESSES_RESULT = asList("Printed Summer Dress", "Printed Summer Dress", "Printed Chiffon Dress");
+    public static final List<String> EVENING_DRESSES_RESULT = Collections.singletonList("Printed Dresses");
+    public static final List<String> WOMAN_RESULT = asList("Faded Short Sleeve T-shirts", "Printed Dress", "Printed Summer Dress", "Printed Summer Dress",
+            "Printed Chiffon Dress", "Printed Dress", "Blouse");
+    public static final List<String> BLOUSE_RESULT = Collections.singletonList("Blouse");
+
+
```
Comment (yuliiakholina) [inline src/main/java/constants/ResultValues.java:25]: pls, delete unnecessary strings

File: src/main/java/search/SearchField.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+import org.jsoup.select.Elements;
+
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+public class SearchField {
+
+    public Response getResponseAfterGet(String searchContent) {
```
Comment (yuliiakholina) [inline src/main/java/search/SearchField.java:17]: method name is not informative

File: src/test/java/search/test/Test.java [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+import java.util.List;
+import java.util.stream.Stream;
+
+import static constants.ResultValues.*;
+import static org.junit.jupiter.params.provider.Arguments.arguments;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class Test {
```
Comment (yuliiakholina) [inline src/test/java/search/test/Test.java:18]: bad class name

File: src/test/java/search/test/Test.java [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    @MethodSource("provideArguments")
+    void verifySearchFieldWithValidData(String searchContent, List<String> list) {
+        SearchField searchField = new SearchField();
+        Response response = searchField.getResponseAfterGet(searchContent);
+        assertEquals(searchField.getFindText(response), searchContent.toLowerCase());
+        assertTrue(searchField.parseFoundElementsToStringCollection(response).containsAll(list));
+        assertEquals(200, response.statusCode());
+    }
+
+    static Stream<Arguments> provideArguments() {
```
Comment (yuliiakholina) [inline src/test/java/search/test/Test.java:31]: should be in separate class like DataProvider\DataUtils

File: src/test/java/search/test/Test.java [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+    @ParameterizedTest
+    @Tag("API")
+    @DisplayName("Verify search field with valid data")
+    @MethodSource("provideArguments")
+    void verifySearchFieldWithValidData(String searchContent, List<String> list) {
+        SearchField searchField = new SearchField();
+        Response response = searchField.getResponseAfterGet(searchContent);
+        assertEquals(searchField.getFindText(response), searchContent.toLowerCase());
+        assertTrue(searchField.parseFoundElementsToStringCollection(response).containsAll(list));
+        assertEquals(200, response.statusCode());
```
Comment (yuliiakholina) [inline src/test/java/search/test/Test.java:28]: excellent test structure

File: src/main/java/constants/ResultValues.java (hunk 1 above)
Comment (4etverochka) [reply src/main/java/constants/ResultValues.java:25]: We receive several dresses with the same name, so test will fail

**Pull request merged**